src/index.html

```diff
@@ -9,32 +9,34 @@
     <h1>Leaderboard</h1>
     <div class="leader-container">
       <section class="recent-scores">
-        <h2>Recent scores</h2>
-        <button class="refresh" type="button">Refresh</button>
-
-        <div class="list-scores" id="scores"></div>
+        <div class="list-header">
+          <h2>Recent scores</h2>
+          <button class="btn" id="refresh" type="button">Refresh</button>
+        </div>
+        <ul class="scores"></ul>
       </section>
       <section class="add-score">
         <h2>Add your score</h2>
-        <div class="form">
+        <form>
           <input
             type="text"
             name="Name"
-            class="name"
+            class="inpute-box"
             id="name"
             placeholder="Your name"
             required
           />
           <input
             type="text"
```

Comment:
## Good job man 🙌🚴‍♂️👏👍🎊🥳🥳👌☕🎉😳🕹

- [ ] [OPTIONAL] One suggestion please change input type to `number` because right now I can also add text as a score. You can see me on the board 🍒

** Happy coding ☕ **

Reply:
ok thanks a lot, ill do that, thanks for the suggestion